locust/dispatch.py

```diff
@@ -98,6 +98,10 @@ def __init__(self, worker_nodes: "List[WorkerNode]", user_classes: List[Type[Use
 
         self._rebalance = False
 
+        self._try_dispatch_fixed = True
+
+        self._no_user_to_spawn = False
```

Comment:
Do you also need to set these in https://github.com/locustio/locust/blob/cce0d29b90e5e9f9dad7a7b96b25a393923c4a16/locust/dispatch.py#L145
?

Reply:
I guess no, **self._no_user_to_spawn** resets to False almost immediately after checking in `_dispatcher`
**self._try_dispatch_fixed** will be True if we stop to spawn somewhere on fixed users, so when we continue it is fine. If we spawned all fixed and it is False, it is also ok. The only case we need to reset it is ramp-down. Also I guess nothing changes if to set this fields, except a superfluous work in _try_dispatch_fixed if it was False.